.github/workflows/shellcheck.yml

```diff
@@ -27,7 +27,7 @@ jobs:
 
     steps:
       # Required to access files of this repository
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@master
```

Comment:
I suggest you use `v3` instead of `master`. `master` branch wasn't updated for years. All development is now happening on the `main` branch.

Reply:
In general I am against using development branches.
In general I would perefer using a released version
that does not change all of a sudden without prior notice.

Reply:
So, in that case, I suggest you use the exact version or pin to SHA1 of the released version. In any case, `master` isn't a good reference.

Reply:
I agree that you don't accept `master` and changed/pinned to `v3`. See releases: https://github.com/actions/checkout/releases